pom.xml

```diff
@@ -42,6 +42,7 @@
 		<spring-cloud-dependencies.version>2021.0.7</spring-cloud-dependencies.version>
 		<spring-cloud-gcp-dependencies.version>${project.parent.version}</spring-cloud-gcp-dependencies.version>
 		<spring-boot-dependencies.version>2.7.13</spring-boot-dependencies.version>
+		<cloud-spanner-r2dbc.version>1.2.2</cloud-spanner-r2dbc.version>
```

Comment:
Where is this used?

Reply:
it was earlier used by `spring-cloud-spanner-spring-data-r2dbc`, not it's unused, so I have removed it